Chinese/Baymax/Baymax.md

```diff
@@ -1 +0,0 @@
-Baymax, 中文译名为大白或者杯面，在电影《超能陆战队》中出现的机器人，主要用于健康检查与辅助。大白非常呆萌和可爱。尤其在女生当中很受欢迎，符合“暖男”形象。
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.